spec/features/bootstrap_cluster_feature_spec.rb

```diff
@@ -85,14 +83,12 @@
       click_on_when_enabled "#bootstrap"
 
       using_wait_time 10 do
-        expect do
-          # means it went to the overview page
-          page.to have_content("Summary")
-          page.to have_content(minions[0].fqdn)
-          page.to have_content(minions[1].fqdn)
-          page.not_to have_content(minions[2].fqdn)
-          page.not_to have_content(minions[3].fqdn)
-        end
+        # means it went to the overview page
+        expect(page).to have_content("Summary")
+        expect(page).to have_content(minions[0].fqdn)
+        expect(page).to have_content(minions[1].fqdn)
+        expect(page).to have_content(minions[2].fqdn)
```

Comment:
@vitoravelino This swapped from .not_to to .to, not sure if that was my change or your changes I merged in.  Do you know which it's supposed to be?  (as mentioned previously it wasn't actually being evaluated before)